### DISCUSSION TITLE
"WAR configuration file not found, using an empty configuration"

### DISCUSSION THREAD

**Discussion by Rock-dev729** [bluenviron/mediamtx#1073] [Questions] [answered]
Ubuntu 20.4
rtsp-simple-server 0.19.3

I've followed the instructions but when I start the rtsp-simple-server I get a "configuration file not found, using an empty configuration" 

I've moved the rtsp-simple-server to /usr/local/bin/
and rtsp-simple-server.yml to /usr/local/etc/

What am I missing?

**Comment by aler9** [comment, marked as answer]
Hello, configuration file is read from the current folder, therefore you can either cd to /usr/local/etc

```
cd /usr/local/etc
rtsp-simple-server
```

or set the path configuration file when launching the server:

```
rtsp-simple-server /usr/local/etc/rtsp-simple-server.yml
```

[a comment by an automated account is not shown]
